src/Literature/Literatures.tsx

```diff
@@ -672,4 +672,51 @@ export const Literatures: Record<LiteratureName, Literature> = {
     filename: LiteratureName.TheSecretWar,
     text: <Typography></Typography>,
   }),
+  [LiteratureName.BeyondCyberpsychosis]: new Literature({
+    title: "Beyond Cyberpsychosis",
+    filename: LiteratureName.BeyondCyberpsychosis,
+    text: (
+      <Typography>
+        Human augmentation has come a long way since the first prosthetics and implants. Now any and all parts of the
+        human body are able to be improved by technology: strength, speed, perception, intelligence. A large variety of
+        powerful augmentations have been developed, but for most they may as well not exist. How did things end up like
+        this?
+        <br />
+        <br />
+        Part of the answer is, unfortunately, that the exact technology that has allowed us to come so far in human
+        augmentation is also responsible for it's great expense and exclusivity.
```

Comment:
```suggestion
        augmentation is also responsible for its great expense and exclusivity.
```
"Its" is the form for possessive, "it's" is only for "it is" or "it has"